### PR TITLE
fix: remove all files related to Goerli

### PR DESCRIPTION
**Pull request by amochuko**
This contain changes that include the deleting all files related to Goerli

## Related GitHub Issue

https://github.com/nation3/citizen-app/issues/290

## How Has This Been Tested?

- [x] Status checks pass (lint, build, test)
- [x] Works on Sepolia preview deployment
- [x] Works on Mainnet preview deployment

## Are There Admin Tasks?

No

### CODE REVIEW

File: ui/components/SwitchNetworkBanner.tsx [hunk 1]
```diff
@@ -4,7 +4,6 @@ import { useNetwork } from '../lib/use-wagmi'
 
 const chainIds = {
   mainnet: 1,
-  goerli: 5,
```
Comment (aahna-ashina) [inline ui/components/SwitchNetworkBanner.tsx:7]: @amochuko Replace `goerli` with `sepolia`

File: ui/lib/config.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ interface Config {
 }
 
 
-const chain = process.env.NEXT_PUBLIC_CHAIN || "goerli";
```
Comment (aahna-ashina) [inline ui/lib/config.ts:33]: @amochuko Replace `goerli` with `sepolia`

File: ui/lib/connectors.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { ethers } from 'ethers'
-import { mainnet, goerli, configureChains } from 'wagmi'
```
Comment (aahna-ashina) [inline ui/lib/connectors.ts:2]: @amochuko Replace `goerli` with `sepolia`

File: ui/lib/connectors.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ import MetaMaskIcon from '../public/icons/connectors/metamask.svg'
 import WalletConnectIcon from '../public/icons/connectors/walletconnect.svg'
 import { networkToId } from './network-id'
 
-const chains = [mainnet, goerli]
```
Comment (aahna-ashina) [inline ui/lib/connectors.ts:12]: @amochuko Replace `goerli` with `sepolia`

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
@amochuko Looks there was some leftover work not completed as part of #261.  And in those cases, we should _replace_ "goerli" with "sepolia" instead of deleting it.

**Comment by amochuko** [comment]
> @amochuko Looks there was some leftover work not completed as part of #261. And in those cases, we should _replace_ "goerli" with "sepolia" instead of deleting it.

oooh, my bad... I'll go fix it now.

**Comment by amochuko** [comment]
> @amochuko Looks there was some leftover work not completed as part of #261. And in those cases, we should _replace_ "goerli" with "sepolia" instead of deleting it.

I have made the fix for it

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**